examples/tsunami/chile2010/setplot.py

```diff
@@ -160,6 +160,29 @@ def add_zeroline(current_data):
     plotaxes.afteraxes = add_zeroline
 
 
+    #-----------------------------------------
+    # Plots of timing (CPU and wall time):
+
+    def make_timing_plots(plotdata):
+        from clawpack.visclaw import plot_timing_stats
+        import os,sys
```

Comment:
Probably can move this to the top of the module as it should not be an issue importing these.